### PR TITLE
NTR: add php minimum check to pipeline

### PR DESCRIPTION
**Pull request by boxblinkracer**
add php minimum check for php 7.2 to plugin and pipelines

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -66,5 +66,9 @@
     "phpstan/phpstan": "^0.12.82",
     "friendsofphp/php-cs-fixer": "^2.18",
     "nunomaduro/phpinsights": "^1.14"
+  },
+  "scripts": {
```
Comment (boxblinkracer) [inline composer.json:70]: here it is 
maybe wew should add a ignore or | true to it for production compatibility?

**Review by barbieswimcrew — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (boxblinkracer) [reply composer.json:71]: solved

File: .github/workflows/ci_pipe.yml [hunk 2]
```diff
@@ -26,13 +26,32 @@ jobs:
...
+    needs: php_checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone Code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
```
Comment (schmitzcarsten) [inline .github/workflows/ci_pipe.yml:42]: wrong php version? Should be 7.2 or am i missing the point here?

File: .github/workflows/pr_pipe.yml [hunk 3]
```diff
@@ -23,13 +23,32 @@ jobs:
...
+    needs: php_checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone Code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
```
Comment (schmitzcarsten) [inline .github/workflows/pr_pipe.yml:39]: should be 7.2?

**Review by schmitzcarsten — CHANGES_REQUESTED** [review]

File: .github/workflows/pr_pipe.yml (hunk 3 above)
Comment (boxblinkracer) [reply .github/workflows/pr_pipe.yml:39]: nope, this is just for dev dependency installation
the check internally is scripted in the makefile and points to php 7.2

**Pull request merged**